opentelemetry-sdk/src/trace/sampler/jaeger_remote/sampler.rs

```diff
@@ -203,7 +203,13 @@ impl JaegerRemoteSampler {
                     // send request
                     match Self::request_new_strategy(&client, endpoint.clone()).await {
                         Ok(remote_strategy_resp) => strategy.update(remote_strategy_resp),
-                        Err(err_msg) => global::handle_error(TraceError::Other(err_msg.into())),
+                        Err(err_msg) => {
+                            otel_error!(
+                                name: "JaegerRemoteSampler.UpdateStrategy.RequestFailed",
+                                message = "Failed to fetch new sampling strategy from remote endpoint. This may cause the sampler to use stale configuration until the next successful update.",
```

Comment:
```suggestion
                                message = "Failed to fetch the new sampling strategy from remote endpoint. This may cause the sampler to use stale configuration until the next successful update.",
```